### PR TITLE
Add Admin Pack for SQL Server chocolatey package

### PR DESCRIPTION
**Pull request by Alea81**
Please suggest better URLs, if you can

### CODE REVIEW

**Comment by Alea81** [comment]
I'm pushing the icon in 5 mins

**Comment by pascalberger** [comment]
> @Alea81 wrote:
> Please suggest better URLs, if you can

Better URLs for what?

File: manual/azure-data-studio-sql-server-admin-pack/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# <img src="https://cdn.rawgit.com/bbtsoftware/chocolatey-packages/19e7388f56f2faf2e2b7a270984d51e93ff8a1cb/icons/vscode-cake.png" width="48" height="48"/> [vscode-cake](https://chocolatey.org/packages/vscode-cake)
```
Comment (pascalberger) [inline manual/azure-data-studio-sql-server-admin-pack/README.md]: Wrong icon and link

**Comment by Alea81** [comment]
- Extension does not have a specific project URL (yet?), the sub extensions do.
- Doc URL is also pointing now to the readme.md
- Does not really have a specific release, or did not find that.
- Icon is now azuredatastudio, should be sql?

File: manual/azure-data-studio-sql-server-admin-pack/tools/chocolateyUninstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Update-SessionEnvironment
```
Comment (pascalberger) [inline manual/azure-data-studio-sql-server-admin-pack/tools/chocolateyUninstall.ps1]: File should be uppercase

File: manual/azure-data-studio-sql-server-admin-pack/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - View Job History with job execution results
+  - Basic Job Control to start and stop jobs
+- [SQL Server Profiler](https://docs.microsoft.com/sql/azure-data-studio/sql-server-profiler-extension?view=sql-server-2017)
+  - Browse through extended events and view associated T-SQL
+  - View and manage sessions
+  - Filter search of events
+- [SQL Server Import](https://docs.microsoft.com/sql/azure-data-studio/sql-server-import-extension?view=sql-server-2017)
+  - Use the Import Flat File Wizard to import .csv and .txt files to a SQL table using AI-based framework to simplify experience
+- [SQL Server dacpac](https://docs.microsoft.com/sql/azure-data-studio/sql-server-dacpac-extension?view=sql-server-2017)
+  - Use the Data-Tier Application Wizard to deploy and extract .dacpac files and import and export .bacpac files
```
Comment (pascalberger) [inline manual/azure-data-studio-sql-server-admin-pack/README.md]: Add a note that this package always installs the latest version of the extension (like done [here](https://github.com/bbtsoftware/chocolatey-packages/blob/master/manual/vscode-cake/vscode-cake.nuspec#L25-L26))

**Comment by pascalberger** [comment]
> Extension does not have a specific project URL (yet?), the sub extensions do.
>  Doc URL is also pointing now to the readme.md
>  Does not really have a specific release, or did not find that.
All are OK for me like that

> Icon is now azuredatastudio, should be sql?
Icon should be the one of the extension, not azure data studio

File: manual/azure-data-studio-sql-server-admin-pack/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# <img src="https://cdn.jsdelivr.net/gh/bbtsoftware/chocolatey-packages@20b4b23cc9798ce6c51f02a8d74d613f948807e9/icons/azure-data-studio-sql-server-admin-pack.png" width="48" height="48"/> [azure-data-studio-sql-server-admin-pack](https://chocolatey.org/packages/azure-data-studio-sql-server-admin-pack)
+
+Admin Pack for SQL Server is a collection of popular database administration extensions to help you manage SQL Server.
+
+## Notes
+
+When you download this pack, you will download the following extensions:
+
+> [!Note]
```
Comment (pascalberger) [inline manual/azure-data-studio-sql-server-admin-pack/README.md]: This is not supported Markdown syntax. Just add the same sentence as done on other packages (see [here](https://github.com/bbtsoftware/chocolatey-packages/blob/master/manual/vscode-cake/vscode-cake.nuspec#L25-L26)) to the list of notes:

> * The package always installs the latest version of the extension.
>    The version of the Chocolatey package reflects not the version of the extension.

**Comment by Alea81** [comment]
@pascalberger ready for review

**Comment by pascalberger** [comment]
Icon is still the one from Azure Data Studio. The one used by the extension is this one: https://raw.githubusercontent.com/Microsoft/vscode-mssql/master/images/sqlserver.png

**Pull request merged**